code-challenges.test.js

```diff
@@ -22,13 +35,63 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+const fiboArray = (num) => {
+  let array = [1,1]
+  for(let i = 2; i < num; i++ ){
+    
+    array.push(array[i - 2] + array[i - 1])
+
+    //array[0] + array[1]
+  }
+  
+  return array
+}      
+
+
 // Pseudo code:
+  //Going to create a function called fiboArray
+  //Function fiboArray will take in a paramater of a number 
+  //Going to declare an array with a starting point of [1,1]
+  //Going to create a for loop letting index be 2 starting point
+  //Use the index to mantain relationship with previous elements
```

Comment:
Line 56 is a little confusing and vague, I would elaborate more on what "maintain relationship with previous elements" means.